frontend/pkg/frontend/ocm.go

```diff
@@ -406,3 +406,29 @@ func (f *Frontend) BuildCSNodePool(ctx context.Context, nodePool *api.HCPOpenShi
 
 	return npBuilder.Build()
 }
+
+// transportFunc implements the http.RoundTripper interface.
+type transportFunc func(*http.Request) (*http.Response, error)
+
+var _ = http.RoundTripper(transportFunc(nil))
+
+func (rtf transportFunc) RoundTrip(r *http.Request) (*http.Response, error) {
+	return rtf(r)
+}
```

Comment:
This `transportFunc` probably belongs in some kind of generic utilities package, as I could see us potentially reusing this.  But we don't have such a package at the moment so this is fine for now.